File: tasks/3190-find-minimum-operations-to-make-all-elements-divisible-by-three.ts
// Accepted 660 / 660 testcases passed Sergey Pomortsev submitted at Jan 28, 2026 20:34  Editorial
// Solution Runtime 0 ms Beats 100.00% Analyze Complexity Memory 57.42 MB Beats 52.90%

const minimumOperations = (nums: number[]): number => nums.filter(el => el % 3).length

File: tasks/3190-find-minimum-operations-to-make-all-elements-divisible-by-three.go
// Accepted 660 / 660 testcases passed Sergey Pomortsev submitted at Jul 05, 2026 13:07
// Runtime 0 ms Beats 100.00% Memory 4.24 MB Beats 13.46%

func minimumOperations(nums []int) int {
  var count int

  for _, num := range nums {
    if num % 3 != 0 {
      count++
    }
  }

  return count
}